connectivity/tests/service.go

```diff
@@ -49,7 +49,9 @@ func (s *podToService) Run(ctx context.Context, t *check.Test) {
 				continue
 			}
 
-			t.NewAction(s, fmt.Sprintf("curl-%d", i), &pod, svc, check.IPFamilyAny).Run(func(a *check.Action) {
+			actionDesc := fmt.Sprintf("curl-%s-%d", check.IPFamilyAny, i)
```

Comment:
For `IPFamilyAny`, I would leave it as is at the moment and not change the output. I personally don't think adding `any` to the name adds value.

Reply:
Okay, I'll fix this.

Reply:
Makes sense. I'll fix this.